### DISCUSSION TITLE
Do I have to accredit DaisyUI when using it in my project

### DISCUSSION THREAD

**Discussion by bouajilaProg** [saadeghi/daisyui#3226] [Q&A] [answered]
Can I use DaisyUI in a project without providing attribution, considering it's for a student club? I'm unsure if the project would be considered commercial or not, and I want to know if I still need to give credit to DaisyUI under these circumstances. If attribution is required, how should it be done properly?

**Comment by saadeghi** [comment, marked as answer]
daisyUI is under MIT license.  
https://github.com/saadeghi/daisyui/blob/master/LICENSE

If you're using it as a NPM dependency, you don't need to do anything because license file is included in the package.

If you want manually copy/paste some files from daisyUI repo and put them in your project, you need to include the daisyUI license file next to that file and make it clear that the file is from daisyUI repo.

No need to provide attribution for the end user.  
You can use it for both free or commercial projects, open source or closed source.

Let me know if you have any questions
